### PR TITLE
Add wfs with HLT as separate step (refactor)

### PR DESCRIPTION
**Pull request by kskovpen**
#### PR description:

Introduce an additional Run 3 wf where the HLT step is separated out from DIGI, following the studies mentioned in https://github.com/cms-sw/cmssw/issues/37564

#### PR validation:

Tested on the newly created 11634.601. 

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Not a backport.

### CODE REVIEW

**Comment by kskovpen** [comment]
test parameters:
- workflow = 11634.601

**Comment by kskovpen** [comment]
please test

**Comment by srimanob** [comment]
test parameters:
- workflows = 11634.601
- relvals_opt = --what upgrade,standard,highstats,pileup,generator,extendedgen,production,ged,machine,premix

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
If the workflow is expected to run every IB, the workflow should be added to `relval2017`.

**Comment by kskovpen** [comment]
> If the workflow is expected to run every IB, the workflow should be added to `relval2017`.

@makortel would you like to have this test wf in IB?

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -3594,6 +3594,15 @@ def gen2021HiMix(fragment,howMuch):
                                       '--eventcontent':'FEVTDEBUGHLT',
                                       '--geometry' : geom
                                       }
+                                      
+    upgradeStepDict['HLTRun3'][k] = {'-s':'HLT:%s'%(hltversion),
+                                 '--conditions':gt,
+                                 '--datatier':'GEN-SIM-DIGI-RAW',
```
[the remark this answers is not in the document]
Comment (kskovpen) [reply Configuration/PyReleaseValidation/python/relval_steps.py:3600]: What about referencing it as SIM-RAW?

**Comment by Dr15Jones** [comment]
please test

**Comment by makortel** [comment]
Thanks @kskovpen. I think running one workflow in IBs would be good.

I'm wondering if it would be better to base this workflow on the `_ProdLike` (.21) workflow, since the idea is to reflect what was done in UL MC production (and so possibly done in production in the future).

**Comment by srimanob** [comment]
A note in case you would like to update the PR, @kskovpen  

Please add a note of the prefix to 
https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/README.md

**Comment by kskovpen** [comment]
Thanks a lot @srimanob and @makortel for your comments.
I would propose to keep this wf at 11634.601. It has not been tested in the production system yet, so let's have it for tests for now and in the meantime we can run some tests in production in order to see if any problem appears (such as the possible data tier adjustment, as mentioned by @srimanob). Once it is fully tested, we can put it in ProdLike. The description has been added in README, as suggested.

**Comment by kskovpen** [comment]
please test

**Comment by kskovpen** [comment]
+pdmv

**Comment by perrotta** [comment]
+1

**Pull request merged**